lite/examples/bert_qa/android/EXPLORE_THE_CODE.md

```diff
@@ -0,0 +1,169 @@
+# TensorFlow Lite BERT QA Android example
+
+This document walks through the code of a simple Android mobile application that
+demonstrates
+[BERT Question and Answer](https://www.tensorflow.org/lite/examples/bert_qa/overview).
+
+## Explore the code
+
+The app is written entirely in Java and uses the TensorFlow Lite
+[Java library](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/java)
+for performing BERT Question and Answer.
+
+We're now going to walk through the most important parts of the sample code.
+
+### Get the question and the context of the question
+
+This mobile application gets the question and the context of the question using the functions defined in the
+file
+[`QaActivity.java`](https://github.com/tensorflow/examples/blob/master/lite/examples/bert_qa/android/app/src/main/java/org/tensorflow/lite/examples/bertqa/ui/QaActivity.java).
+
+
+### Answerer
+
+This BERT QA Android reference app demonstrates two implementation
+solutions,
+[`lib_task_api`](https://github.com/SunitRoy2703/examples/tree/bertQa-android-task-lib/lite/examples/bert_qa/android/lib_task_api)
+that leverages the out-of-box API from the
+[TensorFlow Lite Task Library](https://www.tensorflow.org/lite/inference_with_metadata/task_library/bert_question_answerer),
+and
+[`lib_interpreter`](https://github.com/SunitRoy2703/examples/tree/bertQa-android-task-lib/lite/examples/bert_qa/android/lib_interpreter)
```

Comment:
Use relative paths to avoid dependencies on repo path. eg:

replace - `https://github.com/SunitRoy2703/examples/tree/bertQa-android-task-lib/lite/examples/bert_qa/android/lib_interpreter` with just 
`lib_interpreter` as the current folder is `../examples/bert_qa/android`

Reply:
Thank you for reviewing!

I have replaced the links with relative links